lib/ace/layer/font_metrics.js

```diff
@@ -126,24 +130,12 @@ var FontMetrics = exports.FontMetrics = function(parentEl) {
         }
     };
 
-    this.$measureSizes = function() {
-        if (CHAR_COUNT === 50) {
-            var rect = null;
-            try { 
-               rect = this.$measureNode.getBoundingClientRect();
```

Comment:
removed this because on new browsers clientHeight has same precision as getBoundingClientRect, and the latter reports wrong size for transformed elements